### DISCUSSION TITLE
Edit visuals directly in modV

### DISCUSSION THREAD

**Discussion by TimPietrusky** [vcync/modV#584] [Ideas]
It would be super nice to edit visual code directly in modV to also be able to use it for live coding in combination with other modules. 

For shaders we could integrate https://github.com/patriciogonzalezvivo/glslEditor

**Comment by 2xAA** [comment]
Apart from the realtime controls, which modV already has through props, I'm not sure what glslEditor would bring to modV which isn't already available in a code editor or terminal.

Modules already update on filesystem changes, so editing the code live is already possible.

**Comment by TimPietrusky** [comment]
We actually have this as an issue https://github.com/vcync/modV/issues/619, so we can close this.
